src/main/java/com/rocketden/main/model/Room.java

```diff
@@ -21,4 +28,11 @@
     private String roomId;
 
     private LocalDateTime createdDateTime = LocalDateTime.now();
+
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "host_id")
+    private User host;
+
+    @OneToMany(targetEntity = User.class, cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    private Set<User> users;
```

Comment:
I saw the [link](https://vladmihalcea.com/the-best-way-to-map-a-onetomany-association-with-jpa-and-hibernate/) you shared and it suggested not to have a unidirectional OneToMany mapping. Maybe we could change this to the bidirectional mapping (i.e. remove JoinColumn, add mappedBy to the OneToMany annotation, and add field in User for room) suggested in that article? However, this requires the extra step of adding custom methods to ensure synchronization on both ends, as well as overriding equals and hashCode in some special format.

Another option would be similar to that article's last solution with just a ManyToOne relationship in the User class, and then adding some kind of findUsersByRoom method to UserRepository. 

This is a complex topic so we might need to talk about this in more detail.

Reply:
I think it makes sense to do this -- especially because seeing what `Room` a User is currently a member of would be a useful field. I think I'd prefer to update this in a second PR though; I don't want to bloat this PR too much more, and there are a good number of changes required to fully implement this. I added it as a [Trello task here](https://trello.com/c/jWVjcKix/119-change-users-field-in-the-room-object-to-a-bidirectional-mapping), let me know what you think.

Reply:
Sounds good 👍